template.yml

```diff
@@ -30,8 +30,12 @@ Parameters:
     Type: String
   GitHubUrlParameter:
     Type: String
+    Default: "https://api.github.com"
+    MinLength: 1
```

Comment:
I added a `MinLength` so anyone who had previously deployed with an empty string will fail on deploys until they change it to the default values.